Dockerfiles/agent/README.md

```diff
@@ -26,7 +26,7 @@ The agent is highly customizable, here are the most used environment variables:
 - `DD_SITE`: Destination site for your metrics, traces, and logs. Valid options are datadoghq.com for the Datadog US site, and datadoghq.eu for the Datadog EU site.
 - `DD_HOSTNAME`: hostname to use for metrics (if autodetection fails)
 - `DD_TAGS`: host tags, separated by spaces. For example: `simple-tag-0 tag-key-1:tag-value-1`
-- `DD_CHECK_RUNNERS`: the agent runs all checks in sequence by default (default value = `1` runner). If you need to run a high number of checks (or slow checks) the `collector-queue` component might fall behind and fail the healthcheck. You can increase the number of runners to run checks in parallel
+- `DD_CHECK_RUNNERS`: the agent runs all checks in sequence by default (default value = `4` runners). If you need to run a high number of checks (or slow checks) the `collector-queue` component might fall behind and fail the healthcheck. You can increase the number of runners to run checks in parallel
```

Comment:
I believe this should be rephrased to `the agent runs checks concurrently by default`. We should probably also give a hint there on how to proceed to run them sequentially if necessary, by setting the value back to `1`?